apps/calculator/main.py

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
Please rearrange the imports [as described here](https://zpui.readthedocs.io/en/devel/howto.html#how-to-arrange-imports), and maybe the two empty lines in the beginning are not needed =)